pkg/jobs/worker.go

```diff
@@ -15,6 +15,10 @@ var (
 	defaultMaxExecTime  = 60 * time.Second
 	defaultRetryDelay   = 60 * time.Millisecond
 	defaultTimeout      = 10 * time.Second
+
+	maxMaxExecCount = 5
+	maxMaxExecTime  = 5 * time.Minute
+	maxTimeout      = 1 * time.Minute
```

Comment:
I think that the max limits are a bit too low. Some jobs may have a good reason to have an higher timeout (a big import from a konnector that is ran once a month).